mesonbuild/backend/ninjabackend.py

```diff
@@ -1180,12 +1180,9 @@ def generate_rules(self):
                                 ['$in', '$out'], 'Copying $in to $out'))
 
         c = self.environment.get_build_command() + \
-            ['--internal',
-             'regenerate',
+            ['--reconfigure',
```

Comment:
I think this will raise a warning now, should be `'setup', '--reconfigure'`.

Reply:
Thank you! I dropped to commit for now. Might send it separately, but Idk why the test suite fails.